package.json

```diff
@@ -6,20 +6,23 @@
   "version": "3.12.2",
   "main": "dist/src/index.js",
```

Comment:
for me, tsc build outputted everything under dist directly, so this main entry point should be `dist/index.js` unless there is a version mismatch for where tsc output.
<img width="396" alt="Screenshot 2024-07-02 at 11 17 48 AM" src="https://github.com/radarlabs/react-native-radar/assets/79655657/c8c535b4-1883-4352-82ac-d4749a37385b">